TCMSApp/src/client/app/tests/tests-create.html

```diff
@@ -115,8 +113,10 @@ <h1 class="text-center">Create New Test Case</h1>
           <div class="clearfix">
             <div class="col-sm-3">
               <div class="form-group">
-                <label for="caseAttachment">Attachment:</label>
-                <input type="file" id="caseAttachment" name="caseAttachment">
+                <input type="file" id="caseAttachment" name="caseAttachment" tabindex="-1" style="display:none">
+                <label for="caseAttachment" style="cursor:pointer">
+                  <i class="fa fa-paperclip fa-lg"></i> Attach file
+                </label>
```

Comment:
seems ok. Layout will change so I can understand inline styles now.
(have no chance to check now, but I can believe you that it works) :)